.eslintrc.json

```diff
@@ -1,23 +1,24 @@
 {
```

Comment:
@Thom-IT , avoid changing the settings of .eslintrc.json file done by @IreneRukumbuzi because it's affecting HoundCi